lib/lz4.c

```diff
@@ -124,9 +125,11 @@
 /**************************************
 *  Memory routines
 **************************************/
+#if (HEAPMODE)
```

Comment:
@vvromanov, I recommend you to fix this problem first.  Because currenly [all tests are failed](https://travis-ci.org/Cyan4973/lz4/jobs/131685995) by this problem.
Since [lib/lz4hc.c #includes lib/lz4.c](https://github.com/Cyan4973/lz4/blob/1f47f3f7479c6986b6794f9ad7f13f66ecadc4ff/lib/lz4hc.c#L69), you should also take caring about it.